From 0471972910a009cea1b15886bc083f79a46e5229 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 16 May 2017 15:14:42 +0800
Subject: [PATCH 1795/5242] MLK-14932-1 mmc: sdhci-esdhc-imx: add depends on
 COMPILE_TEST

commit  37aab98111856c30c38a7cc6535ce8bb892c2655 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8 don't define ARCH_MXC, so add COMPILE_TEST to let usdhc driver
also support for i.MX8.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mmc/host/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/Kconfig b/drivers/mmc/host/Kconfig
index 0581c19..e0aca55 100644
--- a/drivers/mmc/host/Kconfig
+++ b/drivers/mmc/host/Kconfig
@@ -200,7 +200,7 @@ config MMC_SDHCI_CNS3XXX
 
 config MMC_SDHCI_ESDHC_IMX
 	tristate "SDHCI support for the Freescale eSDHC/uSDHC i.MX controller"
-	depends on ARCH_MXC
+	depends on ARCH_MXC || COMPILE_TEST
 	depends on MMC_SDHCI_PLTFM
 	select MMC_SDHCI_IO_ACCESSORS
 	help
-- 
1.7.9.5

